04-plots.R

```diff
@@ -8,11 +8,14 @@ library(ggplot2)
 # cited from
 # https://statisticsglobe.com/select-top-n-highest-values-by-group-in-r
 
-top_10_startint <- backtracks_df %>%
+top_10_seq_length <- backtracks_df %>%
   arrange(desc(length)) %>%
-  group_by(length) %>%
-  slice(1:10)
+  slice(1:10) %>%
+  group_by(length)
 
+top_10_startint <- backtracks_df %>%
```

Comment:
Data should be taken from collatz_df, not backtracks_df (due to error in the assignment [ASSIGNMENT.md] - "Creating appropriate graphs that visualise the data wrangling tasks in Task 3.") However, your plots seems to be fine, except Plot 2. In Plot 2, it stated that the red dots represents Top 10, however there are more than 10 red dots.